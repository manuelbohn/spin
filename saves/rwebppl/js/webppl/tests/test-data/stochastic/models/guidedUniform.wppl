var data = [true, true, false, true, true, true];
var model = function() {
  var p = sample(Uniform({a: 0, b: 1}));
  var coin = Bernoulli({p: p});
  factor(sum(map(function(x) {
    coin.score(x);
  }, data)));
  return p;
};
