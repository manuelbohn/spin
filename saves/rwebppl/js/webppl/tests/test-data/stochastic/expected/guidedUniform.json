{
  "mean": 0.75,
  "std": 0.144
}
